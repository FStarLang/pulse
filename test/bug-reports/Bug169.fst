module Bug169

#lang-pulse
open Pulse

// - Failed to desugar pulse extension text
let is_sublist (a b : Seq.seq nat) : prop = true

[@@expect_failure [72]]
fn check_is_sublist (m: SiveT.t)
  requires emp
  ensures emp
{
  ()
}
