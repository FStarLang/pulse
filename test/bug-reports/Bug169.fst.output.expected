>> Got issues: [
* Error 72 at Bug169.fst(10,24-10,29):
  - Module name SiveT could not be resolved
  - Hint: Did you mean Prims, Pulse, FStar.Seq, FStar.Set or FStar.SizeT?

>>]
